### ISSUE TITLE
get picture from library with camera icon

### ISSUE THREAD

**Issue by bjornhjelle** [Biocaching/app#36]
The app has the camera icon at the top. When I click the icon I get two icons at the bottom: one to go to the camera, one to select a picture from the phone library. The second icon does not work.

**Comment by pbb72** [comment]
Indeed, looks like I connected the wrong code to the button. Though, these buttons are a fall-back, you should not have to click them. The camera should come up automatically.
